plugins/core/src/main/kotlin/de/fayard/refreshVersions/core/internal/VersionCatalogs.kt

```diff
@@ -0,0 +1,152 @@
+package de.fayard.refreshVersions.core.internal
+
+import de.fayard.refreshVersions.core.ModuleId
+import de.fayard.refreshVersions.core.internal.UsedPluginsHolder.ConfigurationLessDependency
+import org.gradle.api.artifacts.Dependency
+import org.gradle.api.artifacts.MinimalExternalModuleDependency
+import org.gradle.api.artifacts.VersionCatalog
+import org.gradle.util.GradleVersion
+
+@InternalRefreshVersionsApi
+object VersionCatalogs {
+
+    const val LIBS_VERSIONS_TOML = "gradle/libs.versions.toml"
```

Comment:
the catalog file might not always be named `libs.versions.toml` and there might be multiple files. 

see https://sourcediving.com/manage-your-gradle-dependencies-with-version-catalog-not-only-in-android-114117647cdb#957a

Reply:
Agreed, although this might be good enough for a first version of the versions catalog support